aspnetcore/log-mon/metrics/metrics.md

```diff
@@ -88,15 +87,7 @@ Press p to pause, r to resume, q to quit.
         host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0                                            12
 ```
 
-Run `WebMetric>dotnet-counters list` to show all available metrics.
-
-The following command shows the `Microsoft-AspNetCore-Server-Kestrel` meter with the `requests-per-second` and `total-requests` counters.
-
-```dotnetcli
-dotnet-counters monitor -n WebMetric --counters Microsoft.AspNetCore.Hosting[requests-per-second,total-requests]
-```
```

Comment:
This kind of detail can be left out. People who want to know can look at the dotnet-counters detail page.